jupyter-extension/src/plugins/mount/mount.tsx

```diff
@@ -139,43 +139,43 @@ export class MountPlugin implements IMountPlugin {
         {(_, mounted) => (
           <div className="pachyderm-mount-base">
             <div className="pachyderm-mount-config-container">
-              <div className="pachyderm-mount-base-title">
-                Mounted Repositories
-              </div>
-              <div style={{display: 'flex'}}>
-                <button
-                  className="pachyderm-button-link"
-                  onClick={() => this.setShowPipeline(true)}
+              <div className="pachyderm-mount-base-title">Explore</div>
+              <button
+                className="pachyderm-button-link"
+                data-testid="Datum__mode"
+                onClick={() => this.setShowDatum(true)}
+                style={{
+                  marginRight: '0.25rem',
+                  fontFamily: ['Montserrat', 'sans-serif'],
```

Comment:
The font family should be captured in the classes, also, we probably don't want to change the font family for consistency and cross browser availability.

Reply:
I have moved it to a new css class, though I hesitate the remove it completely. The issue is that the "misalignment" @bbonenfant referred to (which you can also see in the screenshot) is actually a difference in font. Changing the font of this title removes the incongruence.

Let me know what you think. I'll attach "before" and "after" screenshots; I do think it looks better with the matching font. If you think there's a different overall solution that we should use here, though, let me know